frontend/app/components/people-skill-new.js

```diff
@@ -34,10 +34,23 @@ export default Component.extend({
       skills = skills.filter(
         skill => !peopleSkillsIds.includes(skill.get("id"))
       );
-      return skills.sort((a, b) => (a.get("title") < b.get("title") ? -1 : 1));
+      return skills.sort((a, b) => this.orderByTitleCaseInsensitive(a, b));
     });
   }),
 
+  orderByTitleCaseInsensitive(firstSkill, secondSkill) {
+    let firstTitle =
+      firstSkill.get("title") == undefined
+        ? undefined
+        : firstSkill.get("title").toLowerCase();
```

Comment:
You can refactor this to `firstTitle = firstSkill.title && firstSkill.title.toLowerCase();`
This works because at first it tries to call title on firstSkill. If title is `undefined` it will not even try toLowerCase() because it's a falsey value (and thus the && doesn't bother to try the second option because it's already falsey no matter what). If somethings there, it will call the toLowerCase() and return that. That's just JS "magic" that you gotta get used to :)

Also keep in mind that in JS you should ALWAYS compare using `===`. It's not needed anymore after the refactoring but keep it in mind. It's a small error but can lead to big bugs. :upside_down_face: